### DISCUSSION TITLE
Tanker revamp-revamp suggestion

### DISCUSSION THREAD

**Discussion by 11suggestions** [HelixNebulaStudio/RiseOfTheDead#469] [Ideas]
My idea is that we scrap the "Tanker's Rebar", the thing he uses and the actual item currently featured in Dev Branch. Replace it with the sledgehammer. Make him drop that instead of the rebar. 

Make tombs drop "Katana" instead. Tier 1 item, would deal 135 damage per hit but has a very fast swing rate, with a range that's slightly out of a normal zombie's grasp, and would have moderate stamina usage. 

His special attack with the Rebar would be replaced with an attack where he reels up the sledgehammer, creating that red circle you see in Bandit Helicopter and Zomborg Prime to indicate their attacks- quickly expanding to fill a certain distance around him which would be about 1/4 of the room. Then, he'll smash that hammer into the ground and if you're caught in the circle you'll be flung around like his Rebar-spin attack and will be inflicted with Slowed. Or you could just keep the attack we have now and reskin it with the sledgehammer.

**Comment by MrGarrettFerret** [comment]
May I ask what would be the point of adding this?

**Comment by GregTheBestest** [comment]
I don't think its a good idea. Tanker is an early game boss, making him drop the sledgehammer is not great idea.

**Comment by Leroy8888** [comment]
The sledgehammer is a little too strong to include so early in the game. The katana, besides being an oversuggested concept, would also not fit in the game as a tier 1 melee. We already have a machete which is an edged tier 1 melee weapon.

Since Tanker is an early game boss, keeping the boss somewhat simple or straight forward would likely be prefered as the player doesn't have too many options at their disposal yet to deal with complex situations.

**Comment by 11suggestions** [comment]
Thanks for the feedback! Also, this is generally just my opinion on things and how things could be polished. I'm still learning about what I should post here. Sledgehammer could be a hard-mode drop.
